### DISCUSSION TITLE
where'd range comes from?

### DISCUSSION THREAD

**Discussion by rimzzlabs** [Hacker0x01/react-datepicker#3758] [Q&A]
Hello, Im just curious about what is happening here 
https://github.com/Hacker0x01/react-datepicker/blob/df2ac5c6d9a6f6d5298cc3de28f0f95dd8cd5d70/docs-site/src/examples/renderCustomHeader.js#L3

where did the range function comes from?

**Comment by sensologica** [comment]
I would also like to know where the `range` function is imported from.

**Comment by mrKorg** [comment]
import _range from 'lodash/range';
